docs/pages/setup/reference/cli.mdx

```diff
@@ -743,6 +745,58 @@ add_headers:
       "Authorization": "Bearer tokentokentoken"
 ```
 
+#### Aliases
+
+Aliases allow you to define custom commands or change the default flag values for existing commands.
+
+New command `tsh l`:
```

Comment:
I think we should expand this to indicate what happens when you define `aliases.l` (or another key under `aliases`). Do all keys within `aliases` become subcommands of `tsh`? Or does `l` have a special meaning?

Are there any restrictions on aliases? (e.g., multiple words, special characters, etc.)